numpy/core/tests/test_deprecations.py

```diff
@@ -400,5 +400,13 @@ def test_fortran_contiguous(self):
         self.assert_deprecated(np.ones((2,2)).T.view, args=(np.int8,))
 
 
+class TestTestDeprecated(_DeprecationTestCase):
```

Comment:
Ah, if you do this the way you do below (which looks find to me) then you don't need to inherit from `_DeprecationTestCase`.